backbone.js

```diff
@@ -979,14 +979,14 @@
     sort: function(options) {
       if (!this.comparator) throw new Error('Cannot sort a set without a comparator');
       options || (options = {});
+      var comparator = _.isFunction(this.comparator) ? _.bind(this.comparator, this) : this.comparator;
```

Comment:
Passing pre-bound functions will work with Underscore too so that's the compat path I'll point to.